deployments/helm/dra-example-driver/templates/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
```

Comment:
I'm actually not that familiar with `ValidatingAdmissionPolicy`. Can you explain to me in english what this file (in conjunction with [validatingadmissionpolicybinding.yaml](https://github.com/kubernetes-sigs/dra-example-driver/pull/66/files#diff-ae0655a5f1431437c30dc2197f4993c35a566fff2838db7a59992eebcd9a1d62)) is meant to do?

Reply:
I understand it as a set of rules to restrict access from the client (DRA plugin in this case) to the API server. I used [test driver admin-access policy](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/dra/test-driver/deploy/example/admin-access-policy.yaml) as an example.

Reply:
My question is more around what this gives us beyond what we get from the ClusterRole already? At a quick glance it looks as if it also limits each instance of a driver daemonset to only modify the resourceslices on the node it is running on. Is that correct? Is there more to it than that?

Reply:
It also sets a list of acceptable API versions, `v1alpha3` in this particular case.

Reply:
This is  just a selector for what the policy applies to. Once we have v1beta1 and a client uses it, this policy would be ignored.